eslint.config.js

```diff
@@ -0,0 +1,54 @@
+import eslintConfigEslint from "eslint-config-eslint";
```

Comment:
Consistent ESLint capitalization
```suggestion
import eslintConfigESLint from "eslint-config-eslint";
```

Reply:
Fixed in https://github.com/eslint/espree/pull/577/commits/ef5bdf26c758afdd0482c58a68b54a153aead930